packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -350,6 +352,21 @@ class VolumeViewport extends BaseVolumeViewport {
   };
 
   getRotation = (): number => 0;
+
+  /**
+   * Reset the viewport properties to the default values
+   *
+   * @returns void
+   */
+  public resetProperties(): void {
+    this._resetProperties();
+  }
+
+  private _resetProperties() {
```

Comment:
this should have accepted an optional volume id similar to what we have for other apis in the volumeViewport. Please add another PR to fix it. if not provided use the defaultActor 

CC @wayfarer3130 @IbrahimCSAE

Reply:
@sedghi I put up a pull request for this https://github.com/cornerstonejs/cornerstone3D/pull/749/

let me know if it looks good